streams-designer/README.md

```diff
@@ -116,8 +116,8 @@ The first node we'll create is a Python node for loading documents from Cloudant
 
     Name | Type
     --- | --- 
-    id | Text
-    rev | Text
+    _id | Text
```

Comment:
Whilst it is rendering correctly when viewing the markdown file, some stuff is showing up incorrectly (italicized) in the editor because of the leading underscores. I think they need to be escaped.